<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1356</width>
    <height>731</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_subject">
       <attribute name="title">
        <string>Предметы</string>
       </attribute>
       <widget class="QListWidget" name="listWidget_subject">
        <property name="geometry">
         <rect>
          <x>390</x>
          <y>50</y>
          <width>256</width>
          <height>192</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>390</x>
          <y>20</y>
          <width>94</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>Список предметов</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_save">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>90</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Добавить</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_del">
        <property name="geometry">
         <rect>
          <x>480</x>
          <y>260</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Удалить</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>30</y>
          <width>48</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>Название</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_subject">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>60</y>
          <width>291</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_upd">
        <property name="geometry">
         <rect>
          <x>570</x>
          <y>20</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Обновить</string>
        </property>
       </widget>
       <widget class="QGroupBox" name="groupBox_Type">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>90</y>
          <width>116</width>
          <height>125</height>
         </rect>
        </property>
        <property name="title">
         <string>Тип предмета</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QCheckBox" name="cb_lecture">
           <property name="text">
            <string>Лекция</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="cb_seminar">
           <property name="text">
            <string>Семинар</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="cb_laboratory">
           <property name="text">
            <string>Лабораторная</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Расписание</string>
       </attribute>
       <widget class="QTableWidget" name="tableWidget_Schedule">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>9</y>
          <width>1271</width>
          <height>221</height>
         </rect>
        </property>
        <property name="gridStyle">
         <enum>Qt::SolidLine</enum>
        </property>
        <property name="sortingEnabled">
         <bool>true</bool>
        </property>
        <attribute name="horizontalHeaderCascadingSectionResizes">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderCascadingSectionResizes">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderDefaultSectionSize">
         <number>40</number>
        </attribute>
        <column>
         <property name="text">
          <string>Дата</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Время начала</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Время окончания</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Курс</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Номер группы</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Номер подгруппы</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Название предмета</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Тип предмета</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>ФИО преподавателя</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Корпус</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Номер аудитории</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Литера аудитории</string>
         </property>
        </column>
       </widget>
       <widget class="QPushButton" name="btn_refreshShedule">
        <property name="geometry">
         <rect>
          <x>1200</x>
          <y>240</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Обновить</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_addSchedule">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>240</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Добавить</string>
        </property>
       </widget>
       <widget class="QGroupBox" name="groupBox_CreateSchedule">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>280</y>
          <width>731</width>
          <height>281</height>
         </rect>
        </property>
        <property name="title">
         <string>Создание</string>
        </property>
        <widget class="QLabel" name="label_6">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>40</y>
           <width>90</width>
           <height>13</height>
          </rect>
         </property>
         <property name="text">
          <string>Дата проведения</string>
         </property>
        </widget>
        <widget class="QComboBox" name="cb_nameCouple">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>170</y>
           <width>150</width>
           <height>20</height>
          </rect>
         </property>
         <item>
          <property name="text">
           <string>Дискретная математика</string>
          </property>
         </item>
        </widget>
        <widget class="QLabel" name="label_10">
         <property name="geometry">
          <rect>
           <x>610</x>
           <y>40</y>
           <width>90</width>
           <height>13</height>
          </rect>
         </property>
         <property name="text">
          <string>Номер подгруппы</string>
         </property>
        </widget>
        <widget class="QComboBox" name="cb_numGroup">
         <property name="geometry">
          <rect>
           <x>500</x>
           <y>70</y>
           <width>69</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_7">
         <property name="geometry">
          <rect>
           <x>150</x>
           <y>40</y>
           <width>60</width>
           <height>13</height>
          </rect>
         </property>
         <property name="text">
          <string>Номер пары</string>
         </property>
        </widget>
        <widget class="QComboBox" name="cb_numCourse">
         <property name="geometry">
          <rect>
           <x>390</x>
           <y>70</y>
           <width>69</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_endCouple">
         <property name="geometry">
          <rect>
           <x>240</x>
           <y>90</y>
           <width>101</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Конец: </string>
         </property>
        </widget>
        <widget class="QComboBox" name="cb_numSubgroup">
         <property name="geometry">
          <rect>
           <x>610</x>
           <y>70</y>
           <width>69</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="btn_saveSchedule">
         <property name="geometry">
          <rect>
           <x>640</x>
           <y>240</y>
           <width>75</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Применить</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_13">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>210</y>
           <width>80</width>
           <height>13</height>
          </rect>
         </property>
         <property name="text">
          <string>Преподаватель</string>
         </property>
        </widget>
        <widget class="QComboBox" name="cb_typeCouple">
         <property name="geometry">
          <rect>
           <x>180</x>
           <y>170</y>
           <width>91</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_9">
         <property name="geometry">
          <rect>
           <x>500</x>
           <y>40</y>
           <width>71</width>
           <height>13</height>
          </rect>
         </property>
         <property name="text">
          <string>Номер группы</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_startCouple">
         <property name="geometry">
          <rect>
           <x>240</x>
           <y>50</y>
           <width>101</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Начало: </string>
         </property>
        </widget>
        <widget class="QComboBox" name="cb_nameTeacher">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>240</y>
           <width>241</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QDateEdit" name="dateEdit_dateCouple">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>70</y>
           <width>110</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QComboBox" name="cb_timeCouple">
         <property name="geometry">
          <rect>
           <x>150</x>
           <y>70</y>
           <width>69</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_12">
         <property name="geometry">
          <rect>
           <x>180</x>
           <y>140</y>
           <width>70</width>
           <height>13</height>
          </rect>
         </property>
         <property name="text">
          <string>Тип предмета</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_8">
         <property name="geometry">
          <rect>
           <x>390</x>
           <y>40</y>
           <width>63</width>
           <height>13</height>
          </rect>
         </property>
         <property name="text">
          <string>Номер курса</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_11">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>140</y>
           <width>100</width>
           <height>13</height>
          </rect>
         </property>
         <property name="text">
          <string>Название предмета</string>
         </property>
        </widget>
        <widget class="QComboBox" name="cb_dormitory">
         <property name="geometry">
          <rect>
           <x>390</x>
           <y>160</y>
           <width>69</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_14">
         <property name="geometry">
          <rect>
           <x>390</x>
           <y>130</y>
           <width>47</width>
           <height>13</height>
          </rect>
         </property>
         <property name="text">
          <string>Корпус</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_16">
         <property name="geometry">
          <rect>
           <x>590</x>
           <y>130</y>
           <width>47</width>
           <height>13</height>
          </rect>
         </property>
         <property name="text">
          <string>Литера</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_15">
         <property name="geometry">
          <rect>
           <x>490</x>
           <y>130</y>
           <width>56</width>
           <height>13</height>
          </rect>
         </property>
         <property name="text">
          <string>Аудитория</string>
         </property>
        </widget>
        <widget class="QComboBox" name="cb_characher">
         <property name="geometry">
          <rect>
           <x>590</x>
           <y>160</y>
           <width>69</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QComboBox" name="cb_auditorium">
         <property name="geometry">
          <rect>
           <x>490</x>
           <y>160</y>
           <width>69</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="btn_cancelSchedule">
         <property name="geometry">
          <rect>
           <x>540</x>
           <y>240</y>
           <width>75</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Отмена</string>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Аналитические запросы</string>
       </attribute>
       <widget class="QGroupBox" name="groupBox">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>30</y>
          <width>561</width>
          <height>151</height>
         </rect>
        </property>
        <property name="title">
         <string>Среднее количество пар у преподавателя</string>
        </property>
        <widget class="QComboBox" name="cb_teacher">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>50</y>
           <width>231</width>
           <height>21</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>30</y>
           <width>80</width>
           <height>13</height>
          </rect>
         </property>
         <property name="text">
          <string>Преподаватель</string>
         </property>
        </widget>
        <widget class="QDateEdit" name="dateEdit_firstDate">
         <property name="geometry">
          <rect>
           <x>310</x>
           <y>50</y>
           <width>110</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_4">
         <property name="geometry">
          <rect>
           <x>310</x>
           <y>30</y>
           <width>107</width>
           <height>13</height>
          </rect>
         </property>
         <property name="text">
          <string>Первый день недели</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_result1">
         <property name="geometry">
          <rect>
           <x>460</x>
           <y>50</y>
           <width>71</width>
           <height>20</height>
          </rect>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QLabel" name="label_5">
         <property name="geometry">
          <rect>
           <x>460</x>
           <y>30</y>
           <width>53</width>
           <height>13</height>
          </rect>
         </property>
         <property name="text">
          <string>Результат</string>
         </property>
        </widget>
        <widget class="QPushButton" name="btn_calcAvg">
         <property name="geometry">
          <rect>
           <x>460</x>
           <y>100</y>
           <width>75</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Рассчитать</string>
         </property>
        </widget>
       </widget>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>210</y>
          <width>571</width>
          <height>301</height>
         </rect>
        </property>
        <property name="title">
         <string>Показать свободные аудитории</string>
        </property>
        <widget class="QTableWidget" name="tableWidget_FreeAuditorium">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>80</y>
           <width>501</width>
           <height>192</height>
          </rect>
         </property>
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QDateTimeEdit" name="dateTimeEdit_FreeAuditorium">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>40</y>
           <width>194</width>
           <height>22</height>
          </rect>
         </property>
         <property name="calendarPopup">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QPushButton" name="btn_getFreeAuditorium">
         <property name="geometry">
          <rect>
           <x>460</x>
           <y>40</y>
           <width>75</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Показать</string>
         </property>
        </widget>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1356</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
